Sources/Contentful/ArrayResponse.swift

```diff
@@ -107,4 +111,193 @@ public struct MappedArrayResponse<ItemType>: Array where ItemType: EntryModellab
 
     /// The total number of resources which matched the original request
     public let total: UInt
+
+    internal let includes: MappedIncludes?
+
+    internal var includedAssets: [Asset]? {
+        return includes?.assets
+    }
+    internal var includedEntries: [EntryDecodable]? {
+        return includes?.entries
+    }
+}
+
+internal struct MappedIncludes: Decodable {
+    let assets: [Asset]?
+    let entries: [EntryDecodable]?
+
+    private enum CodingKeys: String, CodingKey {
+        case assets     = "Asset"
+        case entries    = "Entry"
+    }
+
+    init(from decoder: Decoder) throws {
+        let container   = try decoder.container(keyedBy: CodingKeys.self)
```

Comment:
Spacing on assignment is not consistent throughout the code. For example, within this same method: https://github.com/contentful/contentful.swift/pull/138/files#diff-0312db99a9cdde535bb9947197e97a8eR148

Either have all of them aligning, or have all of them with proper whitespace. I personally prefer having them with regular whitespacing as it's simpler to read at a glance, and extra whitespace is noisy.

Reply:
i'll see what i can do! thanks @dlitvakb

Reply:
so yeah, i'm simply aligning the right hand side for all these methods to the farthest left that they can be and still remained aligned. make sense?

Reply:
sounds good